scripts/build/clean.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @fileoverview Cleans top-level build directories (for OSS and internal processes),
+ * dist subfolders under packages, and .js[.map] files in package directories.
+ */
+
+const {exec} = require('child_process');
+const fs = require('fs');
+
+const {sync: globSync} = require('glob');
+
+if (require.main === module) {
+  main();
+}
+
+function main() {
+  removeDirectory('build');
+  removeDirectory('.rewrite-tmp');
+  removeFilesOfType('css');
+  removeFilesOfType('js');
+  removeFilesOfType('d.ts');
+  removeFilesOfType('map');
+}
+
+function removeDirectory(directory) {
+  exec(`del-cli ${directory}/ ${directory}`, (err) => {
+    if (err) {
+      throw err;
+    }
+  });
+}
+
+function removeFilesOfType(type) {
+  const fileGlob = `packages/*/*.${type}`;
```

Comment:
Should this be `packages/**/*.${type}` to include sub-directories (e.g., `mdc-drawer/dismissible/`)?

Reply:
yes i just noticed that too when testing text-field. Thanks!